tests/unit/data/test_extract_metadata.py

```diff
@@ -6,90 +6,92 @@
 from edr_server.core.models.metadata import CollectionMetadata
 
 
-class TestExtractMetadata:
-    @staticmethod
-    @pytest.fixture
-    def cube_1():
-        latitude = DimCoord(np.linspace(-90, 90, 4),
-                            standard_name='latitude',
-                            units='degrees')
-        longitude = DimCoord(np.linspace(45, 360, 8),
-                             standard_name='longitude',
-                             units='degrees')
-        time = DimCoord(np.linspace(1, 24, 24),
-                        standard_name='time',
-                        units="hours since 1970-01-01 00:00:00")
-        height = DimCoord(3.5,
-                          standard_name="height",
-                          units="m",
-                          attributes={'positive': 'up'})
-        cube = Cube(np.zeros((4, 8, 24), np.float32),
-                    standard_name="mass_concentration_of_ozone_in_air",
-                    units="ug/m3",
-                    dim_coords_and_dims=[(latitude, 0),
-                                         (longitude, 1),
-                                         (time, 2)])
-        cube.add_aux_coord(height)
-        return cube
+# class TestExtractMetadata:
```

Comment:
The fix for these tests is actually on @tomw66's #36, so once that's merged and this branch is rebased onto the latest main, these tests will pass. So they shouldn't be commented out

Reply:
Oh that's great!  I'll uncomment them and push again.